install/amazon_rds/04_configure_the_collector_docker.mdx

```diff
@@ -47,9 +47,9 @@ Create a new configuration file (e.g. named `pganalyze_collector.replica.env`) a
 DB_HOST=mydbcluster.cluster-ro-123456789012.us-east-1.rds.amazonaws.com
 ```
 
-If you have multiple readers you want to monitor, you either need to add it to
-the content of `CONFIG_CONTENTS`, or run one pganalyze collector Docker
-container for each instance.
+If you have multiple readers with a cluster, you either need to add each instance
```

Comment:
@keiko713 The first part of that sentence reads a bit odd to me - maybe the following would be better?

"If you have a cluster with multiple readers, you either ..."

(same revision in the other files changed)

Reply:
nice, yeah "a cluster with multiple readers" reads better, will update 👍